src/main/java/seedu/address/logic/parser/FindStudentCommandParser.java

```diff
@@ -6,7 +6,7 @@
 
 import seedu.address.logic.commands.FindStudentCommand;
 import seedu.address.logic.parser.exceptions.ParseException;
-import seedu.address.model.student.NameContainsKeywordsPredicate;
+import seedu.address.model.student.StudentNameContainsKeywordsPredicate;
 
 /**
  * Parses input arguments and creates a new FindStudentCommand object
```

Comment:
I think the full stop got removed for some reasons. 

My current updated code base has a full stop for this comment

```suggestion
 * Parses input arguments and creates a new FindStudentCommand object.
```

Reply:
HAHA cause dexter added that in his i think